File: BEE 1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var num = parseInt(lines.shift());
var working_hour = parseInt(lines.shift());
var hour_rate = parseFloat(lines.shift());

console.log("NUMBER =", num);
console.log("SALARY = U$", (working_hour*hour_rate).toFixed(2));

File: BEE 1008.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int num, working_hour;
        float hour_rate;
        num = input.nextInt();
        working_hour = input.nextInt();
        hour_rate = input.nextFloat();
        System.out.println("NUMBER = "+num);
        System.out.printf("SALARY = U$ %.2f\n", working_hour*hour_rate);
    }
}